### ISSUE TITLE
invalid memory address or nil pointer dereference

### ISSUE THREAD

**Issue by cloudrck** [mjtrangoni/flexlm_exporter#76]
### flexlm_exporter version: output of `flexlm_exporter --version`
```
flexlm_exporter, version  (branch: , revision: )
  build user:       
  build date:       
  go version:       go1.19
  platform:         linux/amd64
```

### flexlm_exporter command line flags and relevant YAML configuration file information
```
--path.config="/opt/flexlm_exporter/licenses.yml" \
    --path.lmutil="/opt/flexlm_exporter/lmutil" \
    --log.level=debug
```
### Are you running flexlm_exporter in Docker? 
No

### What did you do that produced an error? 
Run provided binary. Also built 0.0.8 version with Go 1.20 and 1.21. Same issue

### What did you expect to see? 
license data

### What did you see instead?
```
Level=info ts=2023-08-21T21:19:13.839Z caller=tls_config.go:191 msg="TLS is disabled." http2=false
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x28 pc=0x8a93f2]

goroutine 115 [running]:
github.com/mjtrangoni/flexlm_exporter/collector.parseLmstatLicenseInfoServer({0xc000624000, 0x25a, 0x8000?})
        /home/user/app/collector/lmstat.go:231 +0x232
github.com/mjtrangoni/flexlm_exporter/collector.(*lmstatCollector).collect(0xc0000b8c80, 0xc000455f78, 0xc0001006c0?)
        /home/user/app/collector/lmstat.go:422 +0x25c
github.com/mjtrangoni/flexlm_exporter/collector.(*lmstatCollector).getLmstatLicensesInfo.func1({{0xc0002d8034, 0x5}, {0x0, 0x0}, {0xc0000c4768, 0x16}, {0x0, 0x0}, {0x0, 0x0}, ...})
        /home/user/app/collector/lmstat.go:384 +0x85
created by github.com/mjtrangoni/flexlm_exporter/collector.(*lmstatCollector).getLmstatLicensesInfo
        /home/user/app/collector/lmstat.go:381 +0xa9
```

This issue happened after the RHEL Server was patched. As soon as it starts ingesting license data, it throws the memory error.

I've built my own binary using the latest Go, and I've tried to patch go.mod with libraries that had security patches. And I get the same error.

The issue happens when running in a container as well

**Comment by mjtrangoni** [comment]
Hi @cloudrck,

Sorry for the delay. From the time you dont have build information, I guess you are not using `promu` for it.

Did you try building with `make` as it is pointed in the README.md file?

**Comment by cloudrck** [comment]
Yes, this issue appears in both the provided binary. And building my own binary using `make`. For 0.0.8.

Since issue #77 was fixed, using the latest master (or v0.0.9) version does _appear_ not have this issue. But I need to get it pulling data to actually determine

**Comment by mjtrangoni** [comment]
Closing this in favor of #83
